### DISCUSSION TITLE
Date Range Filter for History Section

### DISCUSSION THREAD

**Discussion by Andronovo-bit** [dailydotdev/daily#809] [Feature Requests]
**Description**:
As a daily dev user, I would like to be able to filter my history section based on a specific date range. This would help me to quickly access articles that I read within a certain time frame, without having to manually go through my entire history.

**Solution**:
A date range filter could be added to the history section, allowing users to select a start date and an end date. The history section would then only display articles that were read within the selected date range.

**Benefit**:
This feature would allow users to quickly access articles they read within a specific time frame, making it easier to review and organize their reading history. Additionally, this feature would also allow users to see how much they have been reading on daily dev within a certain time frame, giving them a better understanding of their reading habits.

**Feedback**:
I think this feature would be a great addition to daily dev, making it easier to keep track of my reading history. The ability to filter my history based on a specific date range would make it much more convenient for me to review my reading history and ensure that I never miss a beat. I can't wait to see this feature in action!

**Something like this, but not as bad :D** 😂 

![image](https://user-images.githubusercontent.com/24398592/216815956-89a8a32a-9993-4036-846c-45a20dd0d40e.png)

**Comment by idoshamun** [comment]
Thanks a lot for the suggestion @Andronovo-bit !
